src/core/server/i18n/get_kibana_translation_files.ts

```diff
@@ -17,26 +17,27 @@
  * under the License.
  */
 
-import { KibanaConfig } from '../kbn_server';
-import { fromRoot } from '../../../core/server/utils';
-import { I18N_RC } from './constants';
+import { basename } from 'path';
+import { fromRoot } from '../utils';
 import { getTranslationPaths } from './get_translation_paths';
 
-export async function getKibanaTranslationPaths(config: Pick<KibanaConfig, 'get'>) {
-  return await Promise.all([
+export const getKibanaTranslationFiles = async (
+  locale: string,
+  pluginPaths: string[]
+): Promise<string[]> => {
+  const translationPaths = await Promise.all([
     getTranslationPaths({
       cwd: fromRoot('.'),
-      glob: `*/${I18N_RC}`,
+      nested: true,
     }),
-    ...(config.get('plugins.paths') as string[]).map((cwd) =>
-      getTranslationPaths({ cwd, glob: I18N_RC })
-    ),
-    ...(config.get('plugins.scanDirs') as string[]).map((cwd) =>
-      getTranslationPaths({ cwd, glob: `*/${I18N_RC}` })
-    ),
+    ...pluginPaths.map((pluginPath) => getTranslationPaths({ cwd: pluginPath, nested: false })),
     getTranslationPaths({
       cwd: fromRoot('../kibana-extra'),
-      glob: `*/${I18N_RC}`,
+      nested: true,
     }),
```

Comment:
Are we really still using `kibana-extra`?

Reply:
As long as we still support running plugins from that path then we should keep it. If not there is no need to.

Reply:
We still search for a plugin in `kibana-extra` https://github.com/elastic/kibana/blob/2e37bd070358fa955bc669f9e33d42db4be7f995/packages/kbn-config/src/plugins/plugin_search_paths.ts#L34 and `kbn-pm` supports it as well https://github.com/elastic/kibana/blob/0068c87d6fe7713351acbf42080faa09ff73ef15/packages/kbn-pm/README.md 
But I haven't checked whether it works